custom_components/elastic.py

```diff
@@ -273,8 +287,10 @@ def do_publish(self):
         if not self._only_publish_changed:
             all_states = self._hass.states.async_all()
             for state in all_states:
-                if (state.domain in self._excluded_domains
-                        or state.entity_id in self._excluded_entities):
+                if (not self._included_domains and state.domain in self._excluded_domains) 

```

Comment:
This is complex enough that we should probably pull it out into its own function for readability

Reply:
I rephrased debug messages about skipping entity/domain. The output now only states if a state is skipped or to be published.